airbyte-ci/connectors/pipelines/pipelines/helpers/git.py

```diff
@@ -115,3 +116,15 @@ async def get_modified_files_in_commit(current_git_branch: str, current_git_revi
 def get_modified_files_in_pull_request(pull_request: PullRequest) -> List[str]:
     """Retrieve the list of modified files in a pull request."""
     return [f.filename for f in pull_request.get_files()]
+
+
+@functools.cache
+def get_git_repo() -> git.Repo:
+    """Retrieve the git repo."""
+    return git.Repo(search_parent_directories=True)
+
+
+@functools.cache
+def get_git_repo_path() -> str:
+    """Retrieve the git repo path."""
+    return get_git_repo().working_tree_dir
```

Comment:
💎 
Can we ensure the current git repo is the airbyte one?
We would check the remote url and fail if it's not airbyte?
```
remote_url = repo.remotes[0].config_reader.get("url") 
```